openedx/core/djangoapps/api_admin/forms.py

```diff
@@ -34,11 +35,51 @@ def __init__(self, *args, **kwargs):
         super(ApiAccessRequestForm, self).__init__(*args, **kwargs)
 
 
-class CatalogForm(forms.Form):
-    id = forms.IntegerField(required=False, widget=forms.HiddenInput)
-    name = forms.CharField(required=True, help_text="The name of this catalog")
-    query = forms.CharField(
-        required=True,
-        help_text="The query for courses to be returned by catalog",
-        widget=forms.Textarea
-    )
+class ViewersWidget(forms.widgets.TextInput):
+
+    def render(self, name, value, attrs=None):
+        return super(ViewersWidget, self).render(name, ', '.join(value), attrs)
+
+
+class ViewersField(forms.Field):
+
+    widget = ViewersWidget
+
+    default_error_messages = {
+        'invalid': 'Enter a comma-separated list of usernames.',
+    }
+
+    def to_python(self, value):
```

Comment:
`parse_comma_separated`?
`parse_usernames`?

there must be a better name than `to_python` ;)